web-ui/src/components/guild-results/GuildSummaryCard.jsx

```diff
@@ -143,6 +143,7 @@ const GuildSummaryCard = ({ guild, index, isOpen, onGuildSelect }) => {
         style={{
           color: guild.community ? 'var(--oci-orange)' : 'var(--oci-light-blue)'
         }}
+        // title={`This is a ${guild.community ? 'Community' : 'Guild'}.`}
```

Comment:
This line was recently removed in another PR, but I think we need to add it back to satisfy a story AC that this icon has a tooltip.

Reply:
We could try the MUI tooltip component.

Reply:
@vhscom definitely wasn't suggesting we shouldn't drop an issue somewhere. Just trying to meet the AC without waiting for a fix. 😉 

My 💰 is on MUI being at fault for this one.